Introduction/README.md

```diff
@@ -385,60 +434,40 @@ by the Docker container `fndemouser/gofn:0.0.2` which may be invoked via the
 specified URL.  Now that we've confirmed deployment was successsful, let's
 call our function.
 
-## Calling Your Deployed Function
-
-There are two ways to invoke your deployed function.  The first is using
-the `fn` CLI which makes invoking your function relatively easy.  Type
-the following:
```

Comment:
Might be worth explaining/clarifying that `fn invoke` invokes your function directly independently of any associated triggers, so that you can always invoke a function even without it having any triggers bound to it?

Reply:
Incorporated your thoughts into text.